src/main/kotlin/org/rust/ide/inspections/lints/RsNamingInspection.kt

```diff
@@ -36,10 +36,17 @@ abstract class RsNamingInspection(
         holder.registerLintProblem(
             id,
             "$elementType `$name` should have $styleName case name such as `$suggestedName`",
-            *fixes)
+            *fixes
+        )
     }
 
     abstract fun checkName(name: String): Pair<Boolean, String?>
+
+    companion object {
+        internal val OK = Pair(true, null)
```

Comment:
```suggestion
        @JvmStatic
        protected val OK = Pair(true, null)
```

`internal` modifier doesn't usually make sense in non library code since it's the same as public for current module